jenkins_node_scanner.py

```diff
@@ -231,8 +232,11 @@ def main():
 
                 write_output(args.output_file, node_info)
 
-        logging.debug('Waiting %d seconds', args.period)
-        time.sleep(args.period)
+        if args.period:
```

Comment:
I'd actually appreciate if this logic was inverted:

```
if args.period == 0:
    break


logging.debug('Waiting %d seconds', args.period)
time.sleep(args.period)

Reply:
Hmm, I kinda like this compared to the `if`/`else`. Fixed in 000aa6d.